pgsqlutils/base.py

```diff
@@ -38,6 +38,24 @@ def init_db_conn():
     global engine
     engine = create_engine(conf.DATABASE_URI)
     Session.configure(bind=engine)
+    from . import orm
```

Comment:
Would an `import orm` do the same thing here, given that `.` is the assumed context?